src/common/vars.ts

```diff
@@ -44,6 +45,9 @@ export const issuesTrackerUrl = "https://github.com/lensapp/lens/issues";
 export const slackUrl = "https://join.slack.com/t/k8slens/shared_invite/enQtOTc5NjAyNjYyOTk4LWU1NDQ0ZGFkOWJkNTRhYTc2YjVmZDdkM2FkNGM5MjhiYTRhMDU2NDQ1MzIyMDA4ZGZlNmExOTc0N2JmY2M3ZGI";
 export const supportUrl = "https://docs.k8slens.dev/latest/support/";
 
-const docsVersion = isProduction ? `v${packageInfo.version}` : "latest";
+// This explicitly ignores the prerelease info on the package version
+const { major, minor, patch } = new SemVer(packageInfo.version);
+const mmpVersion = [major, minor, patch].join(".");
```

Comment:
The `v4.2.0` docs might not exist until `4.2.0` is actually released(?) This would mean 404 for `4.2.0-rc.1`. Just show latest?

Reply:
I agree with @jim-docker, pre-releases should point to `latest`.